tests/foreman/api/test_operatingsystem.py

```diff
@@ -103,14 +103,15 @@ def test_positive_create_with_name(self):
 
     @tier1
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1709683)
```

Comment:
@vikask18 , You need not to skip the entire test just for an issue with Debian. 

Make use of bz_bug_is_open, do this in test function:
```
if bz_bug_is_open(1709683):
    if os_family == 'Debian':
        continue
```

Reply:
Done.